nucleus/common/common-util/src/main/java/com/sun/enterprise/util/ProcessExecutor.java

```diff
@@ -81,7 +82,7 @@ public class ProcessExecutor {
      * Creates new ProcessExecutor
      */
     public ProcessExecutor(String[] cmd) {
-        this(cmd, DEFAULT_TIMEOUT_SEC, null);
+        this(cmd, DEFAULT_TIMEOUT_SEC, new String[0]);
```

Comment:
Be careful, this changes the behavior. Maybe replace `== null` with `.length==0`